### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by Milo123459**
Hey 👋,

I've updated all the deps,

I've checked all the docs for breaking changes and everything still works.

I linted the code because standard updated.

:)

### CODE REVIEW

**Comment by PalmerAL** [comment]
This has the same problem as https://github.com/minbrowser/min/pull/1373#issuecomment-744136807, I think? The conclusion from that was that we can't update standard until we finish converting the rest of the old files into modules.

Besides that I think the other updates are fine.

File: package.json [hunk 1]
```diff
@@ -9,27 +9,29 @@
   "dependencies": {
     "dexie": "^3.0.3",
     "dragula": "^3.7.3",
+    "electron-installer-debian": "^3.1.0",
+    "electron-installer-redhat": "^3.2.0",
```
Comment (PalmerAL) [inline package.json:13]: These should only be listed in `optionalDependencies` I think, I'm not sure why npm is moving them (although it did the same thing to me last time I installed).

Comment (Milo123459) [reply package.json:13]: Got it.

**Comment by Milo123459** [comment]
> This has the same problem as [#1373 (comment)](https://github.com/minbrowser/min/pull/1373#issuecomment-744136807), I think? The conclusion from that was that we can't update standard until we finish converting the rest of the old files into modules.
> 
> Besides that I think the other updates are fine.

I mean I'd say set your node version higher :) 14.0.0> should be fine.

**Comment by Milo123459** [comment]
:o This is a lot of conflicts! I'll have to get this fixed! 😰

**Comment by Milo123459** [comment]
@PalmerAL Hey, running yarn start now opens the browser, no errors but I can't press any buttons. Looking into it. BTW, Standard broke like 5 files and I've had to fix them manually.

**Comment by Milo123459** [comment]
Also, standard has formatted everything really wierdly. I'm seriously thinking we should switch to prettier for JS linting.

**Comment by Milo123459** [comment]
Can't open devtools by running CTRL+ALT+SHIFT+I

**Comment by PalmerAL** [comment]
> Standard broke like 5 files and I've had to fix them manually.

This seems bad? If anyone reruns lint, wouldn't they break again? And telling people they have to manually undo some of Standards changes every time they lint isn't a good solution.

I really think that we should just leave the formatting as-is and merge the other dependency updates. The code formatting looks fine to me now after the other PRs, and it's not worth it to constantly reformat everything and break things in exchange for possibly making it look a bit nicer.

**Comment by Milo123459** [comment]
Edit, it was you! (Not trying to throw shade), but I do think we should use prettier for js linting. Reasoning is that it just didn't lint the file.

**Comment by PalmerAL** [comment]
Switching linters again doesn't seem worth it; let's either figure out why Standard is breaking and fix it, or just leave it as-is (which I think is totally fine as a solution, the code is already linted reasonably well).

**Comment by Milo123459** [comment]
Okay, lets leave as is. I'd like to go back to the fact where when I'm starting it, I can't press buttons and can't open dev tools with CTRL+ALT+SHIFT+I

**Comment by PalmerAL** [comment]
Is that with the linting changes, or without? My guess is that the linting probably broke something that prevents those functions from working.

**Comment by Milo123459** [comment]
^^ Nope, it is your module stuff.

**Comment by PalmerAL** [comment]
The app still starts for me, so I'm guessing it's probably not that. Try using the master branch instead of the branch from your PR. Also, you'll need to run `npm install` again to downgrade standard after changing package.json, if you haven't done that already.

**Comment by Milo123459** [comment]
Hmm. Let me try that.

**Comment by Milo123459** [comment]
Already done and same issue.

**Comment by Milo123459** [comment]
Works on master. What a pain.

**Comment by Milo123459** [comment]
this is just annoying me so im closing this

**Pull request closed without merging**